.evergreen.yml

```diff
@@ -292,6 +316,26 @@ tasks:
     commands:
       - func: "clone"
       - func: "test"
+  - name: atlas_e2e
+    commands:
+      - func: "atlas e2e"
+        params:
```

Comment:
these can't be `params` but `env` if you want to be able to use them as environment variables
(`env` is only available in `command: subprocess.exec` type of commands)